content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -30,32 +30,29 @@ your specific needs. The following built-in configuration profiles are currently
 
 1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
 
-1. **sds**: similar to the **default** profile, but also enables Istio's [SDS (secret discovery service)](/docs/tasks/security/citadel-config/auth-sds).
-    This profile comes with additional authentication features enabled by default (Strict Mutual TLS).
-
 1. **remote**: used for configuring remote clusters of a
     [multicluster mesh](/docs/ops/deployment/deployment-models/#multiple-clusters) with a
     [shared control plane](/docs/setup/install/multicluster/shared-vpn/) configuration.
 
+1. **preview**: this profile enables experimental features that may not meet the same level of stability, performance, and security as the default profiles.
+
+1. **empty**: deploys nothing. This can be useful as a base profile for custom configuration.
+
+1. **separate**: this enables the legacy microservices deployment model. This is not recommended, and will be removed in future releases.
```

Comment:
```suggestion
1. **separate**: deploys Istio following the legacy micro-services model. This profile is not recommended because it won't be supported in future releases.
```